README.md

```diff
@@ -43,7 +49,7 @@ AutoGVP Requirements (recommended to place all in the `data/` folder):
 
```

Comment:
for ClinVar VCF, it is now an option for CAVATICA workflow right? 
should we be explicit about that? 

required for custom workflow and optional for CATATICA?